### PR TITLE
added escape-goat for escaping hmtl char entities

### LINKED ISSUE #6998: IDT2 includes on amp - Unescaped characters in alt text
**Describe the bug**
There are unescaped characters in alt text on IDT2 includes on amp, this reads out character by character on a screen reader

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/gahuza/48118437.amp
2. Inspect the alt text for the 1st include
3. You will see unescaped characters in the alt text 
alt=“Afrika y```&#39;```epfo irimwo ubusumbane kurusha ahandi. Gini score.  Ibiharuro vyo mu 2014 kiretse  Namibia (2015) na Amerika (2013).”
4. **&#39;** reads out as  'and number 39' on a screen reader

**Expected behaviour**
Character codes should be ecsaped in alt text

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #6998 

**Overall change:**
_Prevents un-escaped character entities being added to html element attributes (especially with legacy cps content), this uses the escape-goat utility to achieve that_

**Code changes:**

- _Used escape-goat to escape `alt` attribute in IDT2 images  which fixes #6998 ._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added labels to this PR for the relevant pod(s) affected by these changes
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [X] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/utilities/escapeEntities/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// These are the commonly occuring characters, add more if need be.
```
Comment (chris-hinds) [inline src/app/lib/utilities/escapeEntities/index.js]: is it worth us using a pre-built utility for this? Something like `escape-html` is only 335B https://bundlephobia.com/result?p=escape-html@1.0.3

Keen to see what others might think, but using a pre-existing utility would remove the need for maintaining this ourselves as well as removing the tests.

Comment (simonsinclair) [reply src/app/lib/utilities/escapeEntities/index.js]: https://bundlephobia.com/result?p=escape-goat@3.0.0 is also great! 🐐

Comment (rhenshaw56) [reply src/app/lib/utilities/escapeEntities/index.js]: Thanks @hindsc52 @simonsinclair I've changed to use `htmlUnescape` from `escape-goat`. Ran it against the previous test and it's working as expected.

**Review by oluoluoxenfree — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]
Thanks for the changes Rowland 👍

**Review by simonsinclair — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Looks good to me, unescaped characters are skipped while using screen reader..

**Pull request merged**